dev-tools.asciidoc

```diff
@@ -358,10 +358,53 @@ $ npm -g install embark
 ----
 
 === OpenZeppelin
-OpenZeppelin is an open framework of reusable and secure smart contracts in the Solidity language.
-OpenZeppelin integrates well with Truffle
 
-Github link; https://github.com/OpenZeppelin
+https://openzeppelin.org/[OpenZeppelin] is an open framework of reusable and secure smart contracts in the Solidity language.
+
+It is community-driven, lead by the https://zeppelin.solutions/[Zeppelin] developers, with hundreds of contributors from many Ethereum projects. The main focus of the framework is security, achieved by applying industry standard contract security patterns and best practices, taking advantage of all the experience the Zeppelin devs have gained from https://blog.zeppelin.solutions/tagged/security[auditing] a huge amount of contracts, and through the constant testing and auditing from the community that uses the framework as a base for their real world applications.
+
+The OpenZeppelin framework is the most widely used solution for Ethereum smart contracts. This is because the community discusses every proposed solution and learns from the implementation and integration of these solutions, which turns into a growing feedback loop that improves the existing contracts and finds new possibilities to combine them in a clear and secure architecture. It has resulted in the constant addition of new reusable contracts to solve bigger challenges or explore exciting new possibilities on top of the Ethereum blockchain. As mentioned before, the framework currently has an ample library of contracts ranging from implementations of `ERC20` and `ERC721` tokens, to many flavors of crowdsale models, to simple behaviors commonly found in contracts such as `Ownable`, `Pausable` or `LimitBalance`. The contracts in this repository in some cases even function as _de facto_ standard implementations.
```

Comment:
contracts to solve bigger challenges => to solve increasingly complex challenges